src/main/scala/it/pagopa/interop/backendforfrontend/error/BFFErrors.scala

```diff
@@ -72,6 +72,6 @@ object BFFErrors {
       extends ComponentError("0016", s"EService $eServiceId not deletable")
 
   final case class NoDescriptorInEservice(eServiceId: UUID)
-      extends ComponentError("0016", s"No descriptor found in eService $eServiceId")
+      extends ComponentError("0017", s"No descriptor found in eService $eServiceId")
```

Comment:
ℹ️ it's ok for now, but for future PRs: on similar conflicts, with the exception of specific cases, you should update the code of the new PR and not the existing code (in order to maintain already tested functionalities)